app/controllers/application_controller.rb

```diff
@@ -1,6 +1,11 @@
 class ApplicationController < ActionController::Base
+  before_action :configure_permitted_parameters, if: :devise_controller?
 
-    def home
-    end
-    
-end
```

Comment:
I am a little confused about what this end is doing

Reply:
We deleted empty home, and added some protected methods! [Here](https://github.com/CSC322-Grinnell/anti-racism-library/pull/18/files#diff-55c5b7aecfb519d0e4880eaf2788eb6e) is how the whole file looks like